README.md

```diff
@@ -651,29 +651,59 @@ separate section above).
 *form file contracts before running the deploy script. Otherwise the health
 *checks will fail as your node is not ready to upload and download
 
-### Run Docker Command
+### Run Shell Commands
 
-Playbook:
+There are several options to run shell commands against portals via Ansible.
 
-- Run a docker command on portals define in `docker_commands` variable.
+Requires (all options):
 
-Preparation:  
-Defined a `docker_commands` variable in your `portal_versions.yml` file.
+- `shell_commands` variable must be defined as a list of commands to be run.
+- It can be defined either in `my-vars/portal_versions.yml` (e.g. for
+  `portals-deploy` playbook with other variables) or in a separate file e.g.
+  `my-vars/shell-commands.yml`
+
+Depreciation note:
+
+Previously there was `docker_commands` variable with the similar functionality,
+its usage is now depreciated, but for backwards compatibility it works as it
+used to work.
+
+If both `docker_commands` and `shell_commands` are used, commands in both of
+these lists are run (commands from these two lists are added together).
```

Comment:
Do these variables refer to section names in the passed `-e @my-vars/something.yml`? I guess so from the context and examples. It might be useful to state that explicitly for non-savvy ansible users like me who are just doing they best to get stuff done without really knowing a lot about ansible.

Reply:
I think that your question is covered in readme:

- `shell_commands` variable ...
- ... can be defined either in `my-vars/portal_versions.yml` (e.g. for
  `portals-deploy` playbook with other variables) or in a separate file e.g.
  `my-vars/shell-commands.yml`

There is an example code, how to set it inside yaml file
and example code how to run it from shell.